### PR TITLE
Need description about how to move the cluster logging resources for cluster on the vSphere environment

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1788849

### CODE REVIEW

File: modules/infrastructure-moving-logging.adoc [hunk 1]
```diff
@@ -80,3 +80,115 @@ spec:
...
+....
+
+spec:
+
+....
+
+  visualization:
+    kibana:
+      nodeSelector: <1>
+          node-role.kubernetes.io/infra: '' <1>
```
Comment (anpingli) [inline modules/infrastructure-moving-logging.adoc:143]: delete two spaces.
The other LGTM

**Comment by mburke5678** [comment]
Verified in BZ https://bugzilla.redhat.com/show_bug.cgi?id=1788849#c12

**Pull request merged**